feature/system/control_plane_traffic/otg_tests/default_copp_test/README.md

```diff
@@ -0,0 +1,178 @@
+# SYS-1.1: Test default COPP policy thresholds for Arista
+
+## Summary
+
+***NOTE:*** \
+Current verification relies on vendor specific CLI due to the lack of standardized CoPP YANG/OC models. \
+We will not configure any CoPP policies or thresholds on the DUT. We will solely rely on the vendor default and pre-applied CoPP policies and thresholds \
+Vendor implementation may differ and all vendors may not support all the subtests in this readme \
+Use OC wherever applicable and available e.g. configuring interfaces
+[TODO] Utilize standardized OC models as they become available
+
+*   Ensure that the default CoPP protects the control plane from excessive traffic and prioritizes critical control plane packets
+*   Verify that the default CoPP queues function correctly and handle packet drops as expected
+
+* [`featureprofiles/topologies/atedut_2.testbed`](https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_2.testbed)
+
+## Topology:
+
+```mermaid
+graph LR;
+A[ATE] <-- Port1(IPv4/6) --> B[DUT];
+B[DUT] <-- Port2(IPv4/6) --> C[ATE];
+```
+
+## Procedure
+
+### Initial setup
+
+*   Connect DUT port-1, 2 to ATE port-1, 2 respectively
+*   Configure IPv4 and IPv6 addresses on DUT and ATE ports as shown below (using OC)
+
+    *   DUT port-1 IPv4 address ```dp1-v4 = 192.168.1.1/30```
+    *   ATE port-1 IPv4 address ```ap1-v4 = 192.168.1.2/30```
+
+    *   DUT port-2 IPv4 address ```dp2-v4 = 192.168.1.5/30```
+    *   ATE port-2 IPv4 address ```ap2-v4 = 192.168.1.6/30```
+
+    *   DUT port-1 IPv6 address ```dp1-v6 = 2001:DB8::1/126```
+    *   ATE port-1 IPv6 address ```ap1-v6 = 2001:DB8::2/126```
+
+    *   DUT port-2 IPv6 address ```dp2-v6 = 2001:DB8::5/126```
+    *   ATE port-2 IPv6 address ```ap2-v6 = 2001:DB8::6/126```
+
+
+### SYS-1.1.1 - Test CoPP for BGP Traffic
+
+*   Configure BGP between ATE Port-1 and DUT port-1
+*   Generate BGP traffic (update/keepalive) from ATE port-1 within the system defined thresholds
+*   Generate BGP traffic (update/keepalive) from ATE port-1 exceeding the system defined thresholds
+* Monitor CoPP telemetry for the BGP traffic class using the best available method to verify that:
+    *   ***[TODO]*** **gNMI/OC:** Query standardized OC CoPP state model and use OC path via gNMI
+        *   `/components/component/integrated-circuit/pipeline-counters/control-plane-traffic/vendor/..`
+    *   **CLI fallback**: Execute vendor-specific CLI (like show cpu counters queue summary or more specific CoPP counter commands) and parse the output.
+    *   Packets exceeding the threshold are dropped for each queue
+    *   Packets within the threshold are not dropped
+    *   BGP Session remains stable throughout the test
```

Comment:
probably new two sessions on two ports, one which has a good neighbor and a second which has a misbehaving neighbor with too many updates. The good neighbor session should not be affected even when the bad neighbor is rate limited.  The rate limited session may go down, but the good neighbor should stay up.

Reply:
I see, the policing could be implemented system wide (such as a kernel level, iptables style rule filtering for all tcp/179 traffic system wide), so the device could start policing all tcp 179 traffic, impacting all neighbors.  So I agree that we can only reliably have a very simple test which sending a lot of tcp/179 traffic and verifies that the qos scheduler counters (copp/policer counters) increment.

So the change needed for this test is to remove the line "BGP Session remains stable throughout the test", as this is not guaranteed. 

```
/system/control-plane-traffic/ingress/qos/scheduler-policy/scheduler-statistics/scheduler/state/violating-pkts	
/system/control-plane-traffic/ingress/qos/scheduler-policy/scheduler-statistics/scheduler/state/violating-octets	
```